test/reducers/user_revisions.spec.js

```diff
@@ -0,0 +1,27 @@
+import deepFreeze from 'deep-freeze';
+import '../testHelper';
+import user_revisions from '../../app/assets/javascripts/reducers/user_revisions';
+import { RECEIVE_USER_REVISIONS } from '../../app/assets/javascripts/constants';
+
+describe('user revisions reducer', () => {
+  it('should return initial state when no action nor state is provided', () => {
+    const newState = user_revisions(undefined, { type: null });
+    expect(newState).to.be.an('object');
+  });
+
+  it('receives user revisions data with user identification as key via RECEIVE_USER_REVISIONS', () => {
+    const initialState = {
+      title: 'title'
+    };
+    deepFreeze(initialState);
+    const mockedAction = {
+      type: RECEIVE_USER_REVISIONS,
+      data: { course: { revisions: true } },
+      userId: 'username'
+    };
+
+    const newState = user_revisions(initialState, mockedAction);
+    expect(newState.username).to.eq(true);
+    expect(newState.title).to.eq('title');
```

Comment:
Title is not part of this bit of state. The actual initial state is an empty object. If you want to test how it behaves when a second action is received, it would be better to first receive data for one user, than then for a different user, and then expect that the data for the first user is also still present.